languages/luajit/depends

```diff
@@ -0,0 +1 @@
+gcc
```

Comment:
`gcc` is implicit not so?

Reply:
It depends on libgcc, which is why I listed it.

Reply:
I'll remove it, however.

Reply:
Awesome